t/redis.t

```diff
@@ -0,0 +1,37 @@
+use strict;
+use warnings;
+
+use Date::Utility;
+use Test::More;
+use Test::Exception;
+use Data::Chronicle::Writer;
+require Test::NoWarnings;
```

Comment:
Just an obsessive comment! would be better to `use RedisDB` before `require test` so it would seem tidier 🍡

Reply:
sure!

Reply:
Why `require` not `use` here ?

Reply:
@chylli  we were following the same style in other test cases:
- https://github.com/binary-com/perl-Data-Chronicle/blob/master/t/publish.t#L9
- https://github.com/binary-com/perl-Data-Chronicle/blob/master/t/chronicle.t#L11
do you suggest we change them and use `use` instead?

Reply:
I'm only out of curiosity,  want to know the real reason.
If no special reason  I suggest change it to `use` .

Reply:
And of there IS some special reasons, please add a comment.

Reply:
In our case, we have no fixed plan because of `skip_all`, that will cause an error if we `use Test::NoWarnings`'